createdisk.sh

```diff
@@ -10,20 +10,18 @@ SCP="scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_rsa_c
 OC=${OC:-oc}
```

Comment:
From the commit log, I'm not sure I understand the issue there is with CI :) It seems you want to pretend you have a tag when nothing was tagged?